### PR TITLE
Log remote write erros

### PR DESCRIPTION
**Pull request by jwilder**

### CODE REVIEW

File: services/hh/processor.go [hunk 1]
```diff
@@ -137,6 +139,7 @@ func (p *Processor) Process() error {
 
 			// Try to send the write to the node
 			if err := p.writer.WriteShard(shardID, nodeID, points); err != nil {
+				p.Logger.Printf("remote write failed: %v", err)
```
Comment (otoolep) [inline services/hh/processor.go]: Do we need to add to the message that this in the context of hinted-handoff?

**Comment by otoolep** [comment]
Seems sane. +1, with one minor suggestion.

File: services/hh/processor.go (hunk 1 above)
Comment (jwilder) [inline services/hh/processor.go]: There is a `[handoff]` prefix already set on the logger.

Comment (otoolep) [inline services/hh/processor.go]: OK, good enough -- thanks.

**Pull request merged**